modules/dkan/dkan_sitewide/dkan_sitewide.strongarm.inc

```diff
@@ -187,6 +187,18 @@ function dkan_sitewide_strongarm() {
   $strongarm = new stdClass();
   $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
   $strongarm->api_version = 1;
+  $strongarm->name = 'roleassign_roles';
+  $strongarm->value = array(
```

Comment:
Two things:

1. I'd rather not hardcode this with rids, we've been moving toward basing settings like this on role names and setting the variable in the db on install. 

2. We need an install hook for dkan_workflow that will add those roles if workflow is enabled